vfs/directory.go

```diff
@@ -113,13 +116,5 @@ func CreateDirectory(doc *DirDoc, fs afero.Fs, dbPrefix string) error {
 
 	doc.Path = pth
 
-	if err = couchdb.CreateDoc(dbPrefix, doc); err != nil {
-		return err
-	}
-
-	if err = fs.Mkdir(pth, 0755); err != nil {
-		return err
-	}
-
-	return nil
+	return couchdb.CreateDoc(dbPrefix, doc)
```

Comment:
Should we not remove the folder in case of couchdb create error to ensure they stay in sync?